js/modal.js

```diff
@@ -0,0 +1,145 @@
+const works = [
+  {
+    name: 'Tonic',
+    image: '../images/portfolio-img4.jpg',
```

Comment:
- [ ] Your project card images seem not to be displayed on the UI. I think you should remove one `dot` from the image path so that the images can display 👍🏽  

i.e `./images/portfolio-img4.jpg`

![image](https://user-images.githubusercontent.com/63473584/189211728-80aa9e0f-c2c5-4956-aac1-21ee5203878b.png)

![image](https://user-images.githubusercontent.com/63473584/189211814-a8b44cf0-9753-4553-b6cc-c0f8f4739a17.png)